smack-omemo/src/main/java/org/jivesoftware/smackx/omemo/FileBasedOmemoStore.java

```diff
@@ -676,8 +676,9 @@ public CachedDeviceList loadCachedDeviceList(BareJid contact) {
             String[] ids = a.split(",");
 
             for (String id : ids) {
-                if(!id.isEmpty())
+                if(!id.trim().equals("")) {
```

Comment:
I replaced `isEmpty` with `trim().equals("")` because gradle told me, that it doesn't know `isEmpty()` when I ran `gradle check`